test/Microsoft.Azure.SignalR.Tests/Infrastructure/DefaultHubProtocolResolver.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Microsoft.AspNetCore.SignalR.Internal
+{
+    // copied from https://github.com/aspnet/AspNetCore/blob/release/3.0-preview7/src/SignalR/server/Core/src/Internal/DefaultHubProtocolResolver.cs
+    internal class DefaultHubProtocolResolver : IHubProtocolResolver
```

Comment:
Where is this used?

Reply:
cc @anurse

Reply:
Ah tests 😄

Reply:
In UT here, https://github.com/Azure/azure-signalr/blob/dev/test/Microsoft.Azure.SignalR.Tests/ServiceLifetimeManagerFacts.cs#L30